docs/dashboard_config.md

```diff
@@ -124,4 +124,13 @@ spec:
       limits:
         memory: 8Gi
         cpu: '8'
+status:
+  notebookControllerState:
+  - user: username
+      environmentVariables:
+      - key: foo
+          value: bar
+      lastSelectedImage: foo:bar
+      lastSelectedSize: XSmall
+      secrets: odh-secrets-username
```

Comment:
you can safely remove `environmentVariables` and `secrets` fields now because my PR #331 was just merged this morning, which means we don't have these 2 fields on main branch now.
Also, could you align `lastSelectedImage` and `lastSelectedSize` here since they are the same level?

Reply:
Ah good point -- I guess it can be cleaned up now haha.